platform/linux-generic/odp_timer.c

```diff
@@ -844,26 +850,28 @@ static inline void timer_pool_scan_inline(int num, odp_time_t now)
 					odp_atomic_store_u32(&tp->notify_overrun, 2);
 				}
 			}
-			timer_pool_scan(tp, nsec);
+			min = _ODP_MIN(min, timer_pool_scan(tp, nsec));
```

Comment:
Code would be more readable (and robust against macro issues) when a function call is not inside macro. Especially in this case when timer scan is the beef of this function. The min comparison becomes also more readable with variable naming e.g.:

ticks_to_next_expire = timer_pool_scan();
min = MIN(min, ticks_to_next_expire);